### DISCUSSION TITLE
Cannot enable multidiffusion extension no matter what

### DISCUSSION THREAD

**Discussion by IgaMithradir** [vladmandic/sdnext#3345] [Q&A]
I am trying to enable multidiffusion upscaler but when I press the check box and I restart the server nothing happens.
After the restart, the extension is still disabled.
What does the orange circle status mean ?

Thank you

![image](https://github.com/user-attachments/assets/77322f53-7f38-400a-904e-68752d9736a6)

**Comment by brunogcar** [comment]
if you hover over the yellow circle it will show msg that it is supported only by backend original

sdnext has backend diffuser as default, but also supports original as a1111 and its forks 

just switch and use multidiffusion with sd 1.5, 

every time you switch back to diffuser to use sdxl, etc that extension and others that are not designed to support diffusion will be disabled automatically
